checker/src/call_visitor.rs

```diff
@@ -139,11 +137,19 @@ impl<'call, 'block, 'analysis, 'compilation, 'tcx, E>
     /// concrete method that implements the given virtual method and returns the summary of that,
     /// computing it if necessary.
```

Comment:
The method `try_to_devirtualize` now does not return a summary.

Reply:
Yes, it updates the state of the call visitor, or not.